playbook/app/pb_kits/playbook/pb_star_rating/docs/example.yml

```diff
@@ -8,6 +8,7 @@ examples:
     - star_rating_size_options: Size Options
 
   react:
+    - star_rating_interactive: Interactive
```

Comment:
Everything looks great @carloslimasd ! Minor feedback: can we move this doc example to the bottom  so it matches Rails?

Reply:
Yes! I let this be the first to make the test easier, @nidaqg. When we hide the doc, I will do it like Rails.